tests/test_routes.py

```diff
@@ -252,21 +252,21 @@ def test_authorized_project_member(self):
             r = c.get('/texts/urn:cts:formulae:andecavensis.form003/passage/1', follow_redirects=True)
             self.assertTemplateUsed("errors::unknown_collection.html")
             self.assertIn('Angers 3.1' + _(' hat keine Edition.'), r.get_data(as_text=True))
-            c.get('/viewer/urn:cts:formulae:andecavensis.form002.lat001?embedded=True', follow_redirects=True)
-            self.assertTemplateUsed('viewer::multiviewermirador.html')
-            c.get('/viewer/urn:cts:formulae:andecavensis.form002.lat001', follow_redirects=True)
-            self.assertTemplateUsed('viewer::miradorviewer.html')
-            r = c.get('/viewer/urn:cts:formulae:andecavensis.form003?embedded=True', follow_redirects=True)
+            r = c.get('/viewer/urn:cts:formulae:andecavensis.form003', follow_redirects=True)
             self.assertTemplateUsed("errors::unknown_collection.html")
             self.assertIn('Angers 3' + _(' hat keine Edition.'), r.get_data(as_text=True))
-            c.get('/viewer/urn:cts:formulae:andecavensis.form005.lat001', follow_redirects=True)
+            r = c.get('/viewer/urn:cts:formulae:andecavensis.form001', follow_redirects=True)
+            self.assertTemplateUsed('viewer::miradorviewer.html')
+            r = c.get('/texts/urn:cts:formulae:andecavensis.form002.lat001+manifest:urn:cts:formulae:andecavensis.form002.lat001/passage/1+1', follow_redirects=True)
+            self.assertTemplateUsed('main::multipassage.html')
+            r = c.get('texts/manifest:urn:cts:formulae:andecavensis.form002.lat001+manifest:urn:cts:formulae:andecavensis.form001.lat001/passage/1+1', follow_redirects=True)
+            self.assertTemplateUsed('main::multipassage.html')
+            r = c.get('/texts/urn:cts:formulae:andecavensis.form002.lat001+manifest:urn:cts:formulae:andecavensis.form002.lat001/passage/1+all', follow_redirects=True)
+            self.assertTemplateUsed('main::multipassage.html')
```

Comment:
What are the differences between the last three tests that all call `multipassage.html`? If it is only the identifiers that are being called, then you only need a single test.

Reply:
It was useful before the merge of the two branch of r_passage